cookbooks/aws-parallelcluster-config/resources/mount_efs.rb

```diff
@@ -13,13 +13,13 @@
 unified_mode true
```

Comment:
What about naming the resource "manage_efs" instead of "mount_efs" (same for fsx), since its responsibility is not only mounting but also unmounting?

Reply:
I am not in favor of renaming them because they only take care of mount and unmount option without other manage options. The chef `mount` resources also take care of `mount` and `unmount` option.
We use `manage_raid` and `manage_ebs` for ebs and raid because it also include the export and unexport nfs option.

Reply:
Sounds good, I am convinced. Renamed them to `manage_efs`, `manage_fsx`